models/incremental/Intermediate/int_dbt_run_results.sql

```diff
@@ -0,0 +1,47 @@
+{{ config( materialized='incremental', unique_key='command_invocation_id' ) }}
+
+with run_results as (
+    select *
+    from {{ ref('stg_dbt_run_results') }}
```

Comment:
on one line, but with extra lines before and after for symmetrical spacing or whatever